lib/rollbar/item/frame.rb

```diff
@@ -105,7 +105,8 @@ def pre_data(file_lines, lineno)
         to_line = lineno - 2
         from_line = [to_line - MAX_CONTEXT_LENGTH + 1, 0].max
 
-        file_lines[from_line, (to_line - from_line + 1)].select(&:present?)
+
+        file_lines[from_line, (to_line - from_line + 1)].select { |line| line && !line.empty? }
```

Comment:
yah, in this method `file_lines` is an array of strings. I think I did this to not send empty strings on the `pre` section. But fixing this I've seen we are not doing it for the `post`. For now I think this just fixes the issue, but this can be maybe improved.